service/src/main/java/com/derpgroup/derpwizard/manager/AbstractManager.java

```diff
@@ -64,6 +82,31 @@ public void handleRequest(@NonNull VoiceInput voiceInput, @NonNull SsmlDocumentB
     }
   }
 
+  private void appendToConversationHistory(@NonNull VoiceInput voiceInput) {
```

Comment:
Here's the problem, though - the PR after this one changes this data structure from FIFO to LIFO, so any unit tests I built here would immediately not work.  I'll write the tests on that branch and add to that open PR.